tracks/csharp/exercises/sum-of-multiples/README.md

```diff
@@ -0,0 +1,60 @@
+### Reasonable solutions
+
+```csharp
+using System.Linq;
+using System.Collections.Generic;
+
+public static class SumOfMultiples
+{
+    public static int Sum(IEnumerable<int> inputs, int max)
+        => inputs.SelectMany(
+            input => Enumerable.Range(1, (max - 1) / input).Select(numTimes => numTimes * input)
+            , (numtimes, multiple) => multiple).Distinct().Sum();
+}
+```
+
+If the student is not yet ready for / comfortable with full-on Linq then something like the following would be appropriate:
+
+```csharp
+using System.Linq;
+using System.Collections.Generic;
+
+public static class SumOfMultiples
+{
+    public static int Sum(IEnumerable<int> inputs, int max)
```

Comment:
Could you rename `inputs` to `multiples` (which is the parameter's name in the [latest version of the exercise](https://github.com/exercism/csharp/blob/master/exercises/sum-of-multiples/SumOfMultiples.cs))?

Reply:
ditto